src/modules/food.js

```diff
@@ -0,0 +1,26 @@
+import contentLoader from './contentLoader';
+
+const food = (() => {
+  const name = 'Food';
+
+  function switchTo() {
+    const foodArticle = document.createElement('article');
+    foodArticle.className = 'food';
+    foodArticle.innerHTML = `
+      <header class="main">
+        <h1>Odin Restaurant</h1>
+        <h3>Food Menu</h3>
+      </header>
+      <section class="description">
+        <div class="placeholder">
+          Gallery of available food in the restaurant
```

Comment:
Also would be nice to add some actual content to the website to make it more real.

Reply:
Agree